extension/js/common/platform/xss.ts

```diff
@@ -141,24 +144,21 @@ export class Xss {
             a.className = 'image_src_link';
             a.target = '_blank';
             a.innerText = title || 'show image';
-            const heightWidth = `height: ${img.clientHeight ? `${Number(img.clientHeight)}px` : 'auto'}; width: ${
-              img.clientWidth ? `${Number(img.clientWidth)}px` : 'auto'
-            };max-width:98%;`;
             a.setAttribute(
               'style',
               `text-decoration: none; background: #FAFAFA; padding: 4px; border: 1px dotted #CACACA; display: inline-block; ${heightWidth}`
             );
             a.setAttribute('data-test', 'show-inline-image');
             Xss.replaceElementDANGEROUSLY(img, a.outerHTML); // xss-safe-value - "a" was build using dom node api
           } else {
-            img.setAttribute('data-src', img.getAttribute('src') ?? '');
-            img.classList.add('replace_to_base64_image');
-            img.setAttribute('src', '/img/svgs/spinner-green-small.svg');
+            const remoteImgEl = `<div class="remote_image_container" data-src="${src}" data-size="${heightWidth}" data-test="remote-image-container"><span>Authenticity of this remote image cannot be verified.</span></div>`;
```

Comment:
I tested several images and they all have `height: auto; width: auto`, so maybe this calculation doesn't work for remote images?
Also setting original image size for large images will make them too big for email layout.
I think we won't need this `data-size` property here after setting `max-width: 100%` for `.remote_image_container`

Reply:
Well, here's the situation: 
let's say we have a logo.png image that is 300x300 pixels in size. If we style the image to have a width and height of 30 pixels, the rendered image may appear stretched or pixelated.

To avoid this, I think we should  include the original image's size attributes in the HTML code, so that the image is properly scaled down to the desired size.

Reply:
Yes, it'll be nice to have, but as I mentioned I checked a couple remote images and they had `height: auto; width: auto`, which means `img.width ?? img.clientWidth` returns `null`. It seems like we can get dimensions of remote image only after it's loaded.
Also setting `max-width` should automatically resize images according to their proportions, so they won't be stretched or pixelated.

Reply:
image.clientWidth would be null but image.width would be correct value if width is set something like this (like customer's email) `<img src="" width="100"/>`

But I agree. it won't be stretched badly.